src/app/components/Figure/Image/index.jsx

```diff
@@ -1,8 +1,23 @@
+import React from 'react';
 import styled from 'styled-components';
+import { number, string } from 'prop-types';
 
-const Image = styled.img`
+export const StyledImg = styled.img`
   display: block;
   width: 100%;
 `;
 
+export const Img = ({ alt, height, src, width }) => (
```

Comment:
Should this file not also be exporting AmpImg?

Reply:
Yes I think that's what we agreed 👍

Reply:
Yes, this is fixed now.

Reply:
```suggestion
export const Img = ({ alt, src, height, width }) => (
```

(ordering consistency)